Extension/src/main.ts

```diff
@@ -78,6 +78,9 @@ async function offlineInstallation(): Promise<void> {
     setInstallationStage('makeOfflineBinariesExecutable');
     await makeOfflineBinariesExecutable(info);
 
+    setInstallationStage('removeUnnecessaryFile');
+    await removeUnnecessaryFile();
```

Comment:
No.  This is wrong.  There are no unnecessary files in the offline package.  If there are, there is a bug with our Build Definition.